docs/reference/search/rrf.asciidoc

```diff
@@ -516,6 +515,157 @@ We end with `_id: 3` as `_rank: 1`, `_id: 2` as `_rank: 2`, and
+                "description": "rrf score: [0.8333334] computed for initial ranks [2, 1] with rankConstant: [1] as sum of [1 / (rank + rankConstant)] for each query",            <2>
+                "details":                                                                                                                                                        <3>
+                [
+                    {
+                        "value": 2,                                                                                                                                               <4>
+                        "description": "rrf score: [0.33333334], for rank [2] in query at index [0] computed as [1 / (2 + 1]), for matching query with score: ",
+                        "details":                                                                                                                                                <5>
+                        [
+                            {
+                                "value": 0.15876243,
+                                "description": "weight(text:rrf in 0) [PerFieldSimilarity], result of:",
+                                "details":
+                                [
+                                    ...
+                                ]
+                            }
+                        ]
+                    },
+                    {
+                        "value": 1,                                                                                                                                              <6>
+                        "description": "rrf score: [0.5], for rank [1] in query at index [1] computed as [1 / (1 + 1]), for matching query with score: ",
+                        "details":
+                        [
+                            {
+                                "value": 1,
+                                "description": "within top k documents",
+                                "details":
+                                []
+                            }
+                        ]
+                    }
+                ]
+            }
+        }
+        ...
+    ]
+}
+----
+// NOTCONSOLE
+
+<1> the final RRF score for document with `_id=3`
+<2> a description on how this score was computed based on the ranks of this document in each individual query
+<3> details on how the RRF score was computed for each of the queries
+<4> the `value` heres specifies the `rank` of this document in the specific query
+<5> standard `explain` output of the underlying query, describing matching terms and weights
+<6> the `value` heres specifies the `rank` of this document for the second (`knn`) query
+
+In addition to the above, explain in RRF also supports *named queries* using the `_name` parameter.
+This however is not available for retrievers yet, so it is only supported for RRF through the use of  `sub_searches`,
```

Comment:
Do you mean that `retrievers` don't support `_name` yet? Or that they don't support `explain`?

Reply:
IIUC `standard` does support `_name`, whereas `knn` retriever doesn't. For example the following: 
```

GET my_test_index/_search
{
    "retriever": {
        "knn": {
            "field": "my_vector_byte",
            "query_vector": [
                1,
                -2,
                0
            ],
            "num_candidates": 10,
            "k": 10,
            "_name": "my_knn_query"
        }
    }
}
```
returns -> 
```
{
    "error": {
        "root_cause": [
            {
                "type": "x_content_parse_exception",
                "reason": "[12:13] [knn] unknown field [_name]"
            }
        ],
        "type": "x_content_parse_exception",
        "reason": "[12:13] [knn] unknown field [_name]"
    },
    "status": 400
}
```
while for the `standard` retriever, it is parsed w/o any issues: 
```
GET my_test_index/_search
{
    "retriever": {
        "standard": {
            "query": {
                "term": {
                    "body": {
                        "value": "some_value",
                        "_name": "my_term_query"
                    }
                }
            }
        }
    }
}
```
I think that this is because for the `query` value of the `Standard` retriever, we make use of the `AbstractQueryBuilder` which supports `_name`, whereas for `knn` we parse directly all the top level params. 
I was planning on addressing this, but on a separate PR to avoid overcomplicating this one.

Reply:
Added a first take in dada6f6d22a3ccd40bf47d02ca473e0fbcd0016a by adding support for parsing `_name` in the base `RetrieverBuilder` and exposing it for `knn` (which is a top-level query element).